R/n_routine_bloodwork.r

```diff
@@ -52,15 +52,16 @@
 #' additional derived numeric field labelled as "n_routine_bloodwork_derived"
 #'
 #' @export
-#' 
+#'
 
 n_routine_bloodwork <- function(dbcon,
                                 cohort) {
   cohort <- coerce_to_datatable(cohort)
 
-  cat("\n***Note:***
-  The output of this function is based on manual mapping of Sodium and Heamoglobin tests by a GEMINI Subject Matter Expert.
-  Sodium and Hemoglobin are prioritized and mapped for all encounters. However, please carefully check mapping coverage for your cohort of interest, or contact the GEMINI team if you require additional support.\n")
+  mapping_message(
+    "Sodium and Hemoglobin tests",
+    "Note however that Sodium and Hemoglobin are prioritized and mapped for all encounters.\n"
```

Comment:
I think I would just keep the standard message here and remove the additional sentence as it doesn't really add anything.

Reply:
Agree, removed.